recipes/jam/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "jam" %}
+{% set version = "2.6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://swarm.workshop.perforce.com/downloads/guest/perforce_software/jam/jam-{{ version }}.zip
+  sha256: 7c510be24dc9d0912886c4364dc17a013e042408386f6b937e30bd9928d5223c
+  # Removed CC and CFLAG in Makefile 
+  patches: remove_cc.patch
+
+build:
+  number: 0
+  skip: True  # [win]
+  script:
+    - export AR="ar r"  # [unix]
+    - make -j${CPU_COUNT} VERBOSE=1 CC="$CC" CXX="$CXX" CCFLAGS="$CFLAGS" RANLIB="$RANLIB" LINKFLAGS="$LDFLAGS" Cc="${CC}" --environment-overrides BINDIR=$PREFIX/bin   # [unix]
```

Comment:
Is `Cc` a typo or intentional?

Reply:
It is not necessary anymore, it was just for testing purposes, I am going to delete it, thanks for spotting that! :)